### PR TITLE
pace: init at 0.15.2

### PR DESCRIPTION
**Pull request by isabelroses**
## Description of changes

Adds [pace-rs/pace](https://github.com/pace-rs/pace), a tool for time-tracking.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:6]:
```suggestion
```

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:32]:
```suggestion
    meta = {
```
and `lib.` will need to be added below in some spots.
tracking issue: #292468

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:35]: nonexistent license, please choose a license that exists (agpl3Only or agpl3Plus, depending on upstream)

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:17]:
```suggestion
      rev = "refs/tags/pace-rs-v${version}";
```
prevents confusion for update bots esp if an identically-named branch appears

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:21]:
```suggestion
    cargoHash = "sha256-D7jxju2R0S5wAsK7Gd8W32t/KKFaDjLHNZ2X/OEuPtk=";
```

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:33]: nit: makes it clearer that it's a CLI, at least

```suggestion
      description = "Command-line program for mindful time tracking";
```

File: pkgs/by-name/pa/pace/package.nix
Comment (eclairevoyant) [inline pkgs/by-name/pa/pace/package.nix:37]:
```suggestion
```

buildRustPackage already provides an upper bound of supported platforms

**Review by eclairevoyant — CHANGES_REQUESTED** [review]
Thanks for the contribution! Some changes below.

Also, do please rebase such that the maintainer commit is added first on this PR as well

**Review by eclairevoyant — COMMENTED** [review]
Didn't really put the program through its paces (ha) but it appears to run fine. Will approve after the change below.

**Comment by isabelroses** [comment]
Thanks for taking the time to review.

**Review by eclairevoyant — APPROVED** [review]

**Pull request merged**
